lib/appnexusapi/service.rb

```diff
@@ -54,6 +55,19 @@ def get(params={})
     end
   end
 
+  def get_all(params = {})
+    responses = []
+    last_responses = get(params)
+
+    while last_responses.size > 0
+      responses += last_responses
+      last_responses = get(params.merge('start_element' => responses.size))
+      sleep(1) # The gem has no error handling at all for rate limit errors; sleeping for a second prevents errors
```

Comment:
i could change this to 0.7 but not really sure it makes much difference given that it's already sort of a bad answer.